src/fam.rs

```diff
@@ -32,6 +31,14 @@ pub enum Error {
     /// The max size has been exceeded
     SizeLimitExceeded,
 }
+impl std::error::Error for Error {}
+impl fmt::Display for Error {
```

Comment:
nit: can you add newlines between these lines? Helps a bit with readability.
nit2: It would also be nice to mention in the commit message that you are implementing Display because of the Error trait bounds.

Reply:
Added newlines and updated commit message. Fixed?